resources/views/components/ignore-issue.blade.php

```diff
@@ -25,7 +25,7 @@
 >
     <button
         @click="toggleIgnoredUrl('{{$issueUrl}}');"
-        class="flex-1 border text-gray-400 hover:bg-gray-100 dark:border-slate-600 dark:hover:bg-slate-600 font-bold px-3 py-2 ml-1 rounded-md outline-none focus:outline-none ease-linear transition-all duration-150"
+        class="flex-1 btn-secondary"
```

Comment:
I'm cool with abstracting the code out to try and simplify things, but I think I'd prefer the button to be lifted out into a new component that we could use, rather than lifting the CSS out.

Could we please revert this change? I'm cool for you to remake these changes using components in a new PR :)